addons/uh60_flir/functions/fnc_syncAnimation.sqf

```diff
@@ -0,0 +1,21 @@
+/*
+ * Author: Ampersand
+ * Animate the FLIR turret model
+ *
+ * Arguments:
+ * 0: Helicopter <OBJECT>
+ *
+ * Return Value:
+ * 0: Success <BOOLEAN>
+ *
+ * Example:
+ * [_vehicle] call vtx_uh60_flir_fnc_syncAnimation
```

Comment:
![image](https://user-images.githubusercontent.com/4272044/144706993-3ae3da7d-9bc2-4dc6-b8ff-95b80bc4a926.png)
on the MH-60S and the HH-60, the FLIR does not rotate on its own axis, I'm ok with you not fixing it in this PR because it's probably pretty hard to fix, but I just wanted to write it down so you're aware.

Reply:
This animation function also does not work from the copilot seat, this is because the vanilla pilotCamera is not being slewed in the copilot seat.

Reply:
Might need to be fixed in model selections and mempts.

I do see the vanilla pilotcamera being changed from copilot seat though https://youtu.be/6JVG5gBRDwQ